### ISSUE TITLE
Fix Block Type Handle not being present in block queries for GQL

### ISSUE THREAD

**Issue by engram-design** [verbb/vizy#10]

**Comment by pixleight** [comment]
To add more context: data returned from GQL looks like

```json
"rawNodes": [
  {
    "type": "vizyBlock",
    "attrs": {
      "id": "vizy-block-mbIwNPo3uB",
      "enabled": true,
      "collapsed": false,
      "values": {
        "type": "type-IGv2zbLQaK",
```

where `rawNodes.attrs.values.type` is a random string, rather than the block type handle.

**Comment by engram-design** [comment]
Yep, working on revamped GQL querying, which is taking a little longer to implement.

**Comment by engram-design** [comment]
Added much better support for GQL for the next release. I would recommend switching to use `nodes` for this, and a few more benefits, rather than using the `rawNodes`, which is only going to return the data stored by Vizy in the database.

To get this early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.2",
  "...": "..."
}
```

Then run `composer update`.

Also have a look at the updated [docs](https://verbb.io/craft-plugins/vizy/docs/developers/graphql)

**Comment by engram-design** [comment]
Fixed in [1.0.3](https://github.com/verbb/vizy/releases/tag/1.0.3)
